WISE-Community/WISE
# Fixed bug where notebook report was not repopulating

How to test:
1. Log in as a student and launch a project that has the Report enabled
2. Write some work into the Report and click save.
3. Refresh the VLE.
4. Open the Report to see if the work you wrote is repopulated.

Also make sure that you can create and edit regular notes as before.

Closes #1619
